lib/rules/max-attributes-per-line.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @fileoverview Define the number of attributes allows per line
+ * @author Filipa Lacerda
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Define the number of attributes allows per line',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: 'code',
+    schema: [{
+      type: 'object',
+      properties: {
+        'firstline': {
+          type: 'number',
+          minimum: 0
+        },
+        'singleline': {
+          type: 'number',
+          minimum: 1
+        },
+        'multiline': {
+          type: 'number',
+          minimum: 1
+        }
+      }
+    }]
+  },
+
+  create: function (context) {
+    const configuration = context.options[0]
+    const multilineMaximum = configuration.multiline
+    const singlelinemMaximum = configuration.singleline
```

Comment:
@mysticatea 
I am looking at the docs and I am a little lost here. I thought they were assumed by default, meaning, if no options were provided they would fallback to the ones in the schema.

Should I do something like:
`const configuration = context.options[0] || { firstline: 0, singleline: 1, multiline: 1}`
?